src/EditorFeatures/Core/Implementation/AutomaticCompletion/AbstractAutomaticLineEnderCommandHandler.cs

```diff
@@ -84,26 +87,18 @@ public void ExecuteCommand(AutomaticLineEnderCommandArgs args, Action nextHandle
             using (context.OperationContext.AddScope(allowCancellation: false, EditorFeaturesResources.Automatically_completing))
             {
                 // This is a non cancellable command
-                var userCancellationToken = CancellationToken.None;
+                var cancellationToken = CancellationToken.None;
```

Comment:
so strange that this is non-cancellable.  it literally could block indefinitely.  i don't really understand the value of this.